DEA_products/DEA_Wetlands_Insight_Tool.ipynb

```diff
@@ -0,0 +1,2403 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    dc = datacube.Datacube(app="WITbook")
</pre><p>We usually use the exact name of the notebook for "app", e.g. </p><p> </p><p>app="DEA_Wetlands_Insight_Tool"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>